.github/workflows/cleanEnvironments.js

```diff
@@ -0,0 +1,113 @@
+const { Command } = require("commander");
+const fetch = require("node-fetch");
+
+const program = new Command();
+
+program
+  .name("cleanEnvironments")
+  .description("Clean environments")
+  .option("--token <token>", "token fo login to cloud")
+  .option("--snapshot <snapshot>", "snapshot to revert to")
+  .action(async options => {
+    const token = options.token;
+    const snapshot = options.snapshot;
+    const tasksArray = [];
+    const environmentsToClean = await getEnvironmentsForReleaseTesting(token);
+    environmentsToClean.forEach(environment => {
+      tasksArray.push(cleanEnvironment(environment, snapshot, token));
+    });
+    const throwErrorAfterTimeout = setTimeout(function () {
+      throw new Error("Environment didn't upgraded after 30 minutes");
+    }, 1800000);
+    Promise.all(tasksArray).then(tasksList => {
```

Comment:
don't we need to `await` here before `clearTimeout`? Are we sure, that above error will be triggered after 30 minutes, if we will not wait to clear it for promises to be resolved?